src/blueprints/tv2/timeline-object-factories/tv2-atem-video-mixer-timeline-object-factory.ts

```diff
@@ -72,7 +72,7 @@ export class Tv2AtemVideoMixerTimelineObjectFactory implements Tv2VideoMixerTime
   }
 
   public createUpstreamKeyerTimelineObject(downstreamKeyer: Tv2DownstreamKeyer, enable: TimelineEnable): AtemMeTimelineObject {
-    const downstreamKeyerNumber: number = downstreamKeyer.Number + 1
+    const downstreamKeyerNumber: number = downstreamKeyer.name + 1
```

Comment:
I considered it.
I ended up keeping it since the type was still `number`, but then you could argue that `downstreamKeyer.name` shouldn't be called `name` either.
Don't have any strong preferences here, so whatever the majority is up for.